docs/lake/design.md

```diff
@@ -495,11 +499,11 @@ data during any key-range scan.
 ### Delete
 
 Data objects can be deleted with the `delete` command.  This command
-simply removes the data from branch without actually deleting the
+simply removes the data from the branch without actually deleting the
 underlying data objects thereby allowing time travel to work in the face
 of deletes.
 
-For example, this command deletes the three objects and/or commits referenced
+For example, this command deletes the three objects referenced
```

Comment:
We discussed this change on Slack. The help text still speaks as if `delete` would accept commits, so I mentioned that in #3012 along with the request to add a test for it.